frontend/pulse3d/src/components/interactiveAnalysis/WaveformGraph.js

```diff
@@ -143,8 +143,24 @@ export default function WaveformGraph({
   openChangelog,
   undoLastChange,
   peakValleyWindows,
-  setPeakValleyWindows,
   checkDuplicates,
+  twentyFourPlateDefinition,
+  peakSlope,
+  setPeakSlope,
+  peakYIntercept,
+  setPeakYIntercept,
+  peakY1,
+  setPeakY1,
+  peakY2,
+  setPeakY2,
+  valleySlope,
+  setValleySlope,
+  valleyYIntercept,
+  setValleyYIntercept,
+  valleyY1,
+  setValleyY1,
+  valleyY2,
+  setValleyY2,
```

Comment:
this seems like way too many props to pass down? is there a way to condense these?